tendermint/src/amino_types/message.rs

```diff
@@ -15,6 +17,20 @@ pub trait AminoMessage: prost_amino::Message {
         self.encode(&mut res).unwrap();
         res
     }
+
+    /// Encode prost-amino message as length delimited.
+    ///
+    /// Warning: Only use this method, if you are in control what will be encoded.
+    /// If there is an encoding error, this method will panic.
+    fn bytes_vec_length_delimited(&self) -> Vec<u8>
+    where
+        Self: Sized,
+    {
+        let len = self.encoded_len();
+        let mut res = Vec::with_capacity(len + encoded_len_varint(len as u64));
```

Comment:
Awesome, thanks!

Why did you decide against `len.try_into().unwrap()` (as suggested in https://github.com/interchainio/tendermint-rs/pull/69#discussion_r347431877) instead of `len as u64`? I think, the latter can silently overflow while the first would panic.

Reply:
Nice trick, re-commited.

Reply:
Awesome, thanks!